pycbc/events/stat.py

```diff
@@ -1290,8 +1291,11 @@ def __init__(self, sngl_ranking, files=None, ifos=None, **kwargs):
             construct the coincident statistics. The files must have a 'stat'
             attribute which is used to associate them with the appropriate
             statistic class.
-        ifos: list of strs, not used here
+        ifos: list of strs
             The list of detector names
+        reference_ifos: comma separated list of ifo prefixes
```

Comment:
The default is not a list, it's a string.

Reply:
fixed